code/client/include/Client.h

```diff
@@ -0,0 +1,29 @@
+#ifndef COLLATZ_CONJECTURE_CLIENT_H
+#define COLLATZ_CONJECTURE_CLIENT_H
+
+#include <yarp/os/all.h>
+#include <vocabs.hpp>
+#include <ClientIDLInterface.h>
+class Client : public yarp::os::RFModule, ClientIDLInterface{
+private:
+
+    yarp::os::Port rpcPort;
+    std::string rpcPortName;
+    yarp::os::RpcClient clientPort;
+    std::string clientPortName;
+    int testOutcome;
+    int n;
+    int t;
+    void collatz_test(int n, int t);
```

Comment:
Don't forget that in **C++** we make intensive use of **`const` attribute**, even for tiny classes such as `int`.